airframe-http-recorder/src/main/scala/wvlet/airframe/http/recorder/HttpRecorder.scala

```diff
@@ -96,20 +93,11 @@ object HttpRecorder extends LogSupport {
 
   private def newDestClient(recorderConfig: HttpRecorderConfig): Service[Request, Response] = {
     debug(s"dest: ${recorderConfig.destAddress}")
-    val clientBuilder =
-      ClientBuilder()
-        .stack(Http.client)
-        .name("airframe-http-recorder-proxy")
-        .dest(recorderConfig.destAddress.hostAndPort)
-        .noFailureAccrual
-        .keepAlive(true)
-        .retryPolicy(RetryPolicy.tries(3, RetryPolicy.TimeoutAndWriteExceptionsOnly))
 
-    (if (recorderConfig.destAddress.port == 443) {
-       clientBuilder.tls(SSLContext.getDefault)
-     } else {
-       clientBuilder
-     }).build()
+    Finagle.client
+      .withInitializer(_.withLabel("airframe-http-recorder-proxy"))
+      .newClient(recorderConfig.destAddress.hostAndPort)
+      .nativeClient
```

Comment:
This change was necessary to properly configure SSL. This also avoids using the deprecated stack builder.